### ISSUE TITLE
Save seccion in storage external 

### ISSUE THREAD

**Issue by rodolfovizcay** [orkestral/venom#2486]
Dear All. Sorry, I know that perhaps it is a well-known topic but the link in the documentation is broken and in the npm documentation there is no mention of how to do it. 

* How can you save the section to external storage as JSON or other media to retrieve it later without the need to read the QR again?

**Comment by abhilash-screlinfo** [comment]
which platform are you running the bot in?
there is a parameter that you can set on where the session data to be saved.

-----------------------------------------------------------------------------------------------------------------------------------
venom.create(
  id,
  {
    useChrome: true, // If false will use Chromium instance
    logQR: true, // Logs QR automatically in terminal
    disableSpins: true, // Will disable Spinnies animation, useful for containers (docker) for a better log
    disableWelcome: true, // Will disable the welcoming message which appears in the beginning
    updatesLog: true, // Logs info updates automatically in terminal
    autoClose: 120000, // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)
    folderNameToken: '/home/user/WAtokens/tokens', // Automatically closes the venom-bot only when scanning the QR code (default 60 seconds, if you want to turn it off, assign 0 or false)
  },
).then((client) => {});
-----------------------------------------------------------------------------------------------------------------------------------
above is a code example in venom-bot run in ubuntu server 22.04
put the location where you want as the value of the key  'folderNameToken'
since i haven't tested this in windows i hope it works the same way.

**Comment by rodolfovizcay** [comment]
Thanks for the help.

The tokens folder is created, but I want to recover the section with the same name when the server restarts.

How can we save that section (client) so that when the server shuts down or has a memory leak it can be picked up again without reading the QR again?

**Comment by abhilash-screlinfo** [comment]
When the next time server startup after an outage or a shutdown/reboot
you should call the same session create function (venom.create()) with the same id, venom-bot automatically checks if it is already logged ie, it automatically checks if the old login is present in the location you have given in the  'folderNameToken' value. If the tokens saved from the previous login is present in there bot will login automatically from that json file.
You dont have to scan the QR again.

TLDR : venom will automatically reload the json from the location given that you pass the same id you used when you created the session previously.
